inference-engine/src/mkldnn_plugin/mkldnn_exec_network.cpp

```diff
@@ -98,10 +98,10 @@ MKLDNNExecNetwork::MKLDNNExecNetwork(const InferenceEngine::ICNNNetwork &network
                 i++;
             }
 
-            if (with_cpu_x86_bfloat16() && isFloatModel) {
+            if (_cfg.enforceBF16 && isFloatModel) {
                 BF16Transformer bf16Transformer;
                 CNNNetwork cnnetwork(_clonedNetwork);
-                if (cfg.enforceBF16 == true) {
+                if (_cfg.enforceBF16 == true) {
```

Comment:
I think this condition becomes redundant right now.

Reply:
I believe we also discussed the case of models with manually marked BF16 edges. For this kind of models behavior should be the following:
1. If (enforceBF16 == true) than call convertToFloat and after that call convertToBFloat16
2. If (enforceBF16 == true) just don't modify the model at all.

Reply:
Done